pytest_fixtures/component/rh_cloud.py

```diff
@@ -36,10 +36,16 @@ def module_target_sat_insights(request, module_target_sat, satellite_factory):
         iop_settings = settings.rh_cloud.iop_advisor_engine
         script = (iop_settings.setup_script or '').splitlines()
 
+        http_proxy = (
+            f'HTTPS_PROXY={settings.http_proxy.http_proxy_ipv6_url} '
+            if settings.server.is_ipv6
+            else ''
+        )
+
```

Comment:
I'd suggest using `enable_ipv6_system_proxy` helper here instead to configure the proxy

Reply:
Fixed.